user-guide/Reference/DataMiner_Extension_Modules.md

```diff
@@ -4,12 +4,19 @@ uid: DataMinerExtensionModules
 
 # DataMiner Extension Modules (DxMs)
 
-A DataMiner Extension Module (DxM) is a service that can be installed, upgraded and uninstalled without the need to reboot the DataMiner Agent. These modules will expand the core DataMiner software with additional features and capabilities.
+A DataMiner Extension Module (DxM) is a service that can be installed, upgraded and uninstalled without the need to reboot the DataMiner Agent. These modules will expand the core DataMiner software with additional features and capabilities. 
+
+# Evolution
+
+From the list of DxMs, we will extract some modules that are actually core modules created with the DXM technology and call them 'DMs'. DxMs and DMs will be treated differently during upgrades. DMs will be installed (or updated if already installed) during Core upgrade/update packages, while DxMs will only be updated when they were already installed. 
+DxMs will have a 'minimal version of the core' dependency, and that will be made transparent through the cloud admin page. When installing updates, the latest compatible update will be automatically selected.
 
 ## Packs
 
 For ease of installation, DxMs may be packaged together in a pack, such as the [Cloud Pack](xref:CloudPackages). These packs allow users to install several DxMs at once.
 
+## Available DMs
```

Comment:
Are any DMs available yet? If so, could you add these below this title? I'd prefer not to have a dummy section without any content here, as that may look quite confusing.